Developer-Toolbox/Models/WeeklyChallenge.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.ComponentModel.DataAnnotations;
+namespace Developer_Toolbox.Models
+{
+    public class WeeklyChallenge
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required(ErrorMessage = "The title field is required!")]
+        [StringLength(100, ErrorMessage = "The title cannot exceed 100 characters")]
+        [MinLength(5, ErrorMessage = "The title must be at least 5 characters long")]
+        public string Title { get; set; }
+        [Required(ErrorMessage = "The description field is required!")]
+        public string Description { get; set; }
+
+        [Required(ErrorMessage = "Difficulty is required.")]
+        [Range(1, 10, ErrorMessage = "Difficulty must be between 1 and 10.")]
+        public int Difficulty { get; set; }
```

Comment:
Let's change here so as to have the same format of the difficulty level everywhere in the app (the difficulty should be a string: Easy, Intermediate, Difficult; the exact names are found in an enum)

Reply:
Thank you for your suggestion! I've updated the code to use the enum values for difficulty levels (Easy, Intermediate, Difficult) as a string, ensuring consistency across the app. The change is now reflected in the model.